content/en/tracing/troubleshooting/iis_dotnet_framework.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Troubleshooting - Not seeing .NET Framework traces
+kind: documentation
+---
+
+## Problem 
+
+.NET Framework APM traces are not showing within Datadog.
+
+## Solution
+
+If .NET Framework APM traces are not showing within Datadog, please review the possible reasons below.
+
+### Compatability 
+
+The .NET Tracer supports automatic instrumentation on .NET Framework 4.5 and above. 
+
+
+### Confirm Installation 
+
+#### Datadog Agent
+
+Has the Datadog Agent for Windows OS been installed locally? If not, please visit our [Datadog Windows agent][1] guide.
+
+#### Datadog .NET Tracer 
+
+Has the Datadog .NET tracer been installed locally? If not, please visit our [Datadog .NET APM release page][2].
+
+### Automatic Instrumentation 
+
+#### Verify Required Environment Variables
+
+For Automatic Instrumentation for IIS application(s), the required environment variables must be set within the `W3SVC` & `WAS` Windows Registry paths. 
```

Comment:
```suggestion
For automatic instrumentation of IIS applications, the required environment variables must be set within the `W3SVC` & `WAS` Windows registry paths. 
```

Reply:
I think "Windows Registry" should remain capitalized since it's is a product name.
https://en.wikipedia.org/wiki/Windows_Registry